### PR TITLE
Added how to get contract addresses on a development node

### PR DESCRIPTION
**Pull request by qbzzt**

### CODE REVIEW

File: src/docs/developers/l2/dev-node.md [hunk 1]
```diff
@@ -182,3 +182,18 @@ If you'd like to follow these logs as they're being generated, run:
 ```sh
 docker logs --follow <name of container>
 ```
+
+
+### Getting contract addresses
+
+The [Optimistic Ethereum contracts](/docs/protocol/protocol.html#chain-contracts) 
+are already deployed on the development nodes. The contracts on L2 always have the 
+addresses same, so you can
```
Comment (K-Ho) [inline src/docs/developers/l2/dev-node.md:191]:
```suggestion
same addresses, so you can 
```

**Review by K-Ho — COMMENTED** [review]
one small nit - also this is probably not the ideal way to get L1 contract addresses locally. We should really include the contract addresses for local setup here as well https://github.com/ethereum-optimism/optimism/tree/develop/packages/contracts/deployments

**Comment by qbzzt** [comment]
> This is probably not the ideal way to get L1 contract addresses locally. We should really include the contract addresses for local setup here as well https://github.com/ethereum-optimism/optimism/tree/develop/packages/contracts/deployments

Do all local setups have the exact same L1 contract addresses? Do those addresses stay fixed when we change something in the settings? Unless the answer to both questions is yes, we need to teach people how to get them rather than publishing them ourselves.

**Comment by K-Ho** [comment]
The addresses should be fixed unless we make protocol changes (e.g. add a new redeploy)  - when we do make changes like that, the markdown ought to be auto updated

**Review by K-Ho — APPROVED** [review]

**Pull request merged**